### PR TITLE
Use GitHub Action for CI, add Ruby 2.7 and 3.0, drop Ruby 2.4

### PR DESCRIPTION
**Pull request by dadah89**
cc @grosser

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7', '3.0']
+        activerecord: ['4.2', '5.0', '5.1', '5.2']
+        task: ['bundle exec rake spec']
```
Comment (grosser) [inline .github/workflows/actions.yml:14]:
```suggestion
```

Comment (grosser) [reply .github/workflows/actions.yml:14]: remove all mentions of task and hardcode spec
atm it runs `bundle exec rake bundle exec rake spec` ... idk how that ever worked :D

**Review by grosser — APPROVED** [review]

**Comment by dadah89** [comment]
@grosser fixed the task references. I'm not seeing the GA running on this PR though, is it something that needs to be enabled in the repo settings?

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7', '3.0']
+        activerecord: ['4.2', '5.0', '5.1', '5.2']
```
Comment (bquorning) [inline .github/workflows/actions.yml:13]: There are only two gemfiles, right?
```suggestion
        activerecord: ['4.2', '5.2']
```

**Comment by grosser** [comment]
first merge will enable it ... it's a bit yanky

**Comment by grosser** [comment]
just make a new PR to fix it https://github.com/grosser/active_record-comments/actions/runs/486321687

**Review by bquorning — COMMENTED** [review]
> just make a new PR to fix it

I can fix it. I've been migrating repositories from Travis to GHA for the last two days anyway 😅

**Comment by dadah89** [comment]
@grosser @bquorning removed wwtd and fixed the action in https://github.com/grosser/active_record-comments/pull/15

**Comment by dadah89** [comment]
@bquorning oh I see, it needs more fixing. Guess I'll just wait for https://github.com/grosser/active_record-comments/pull/14 then :)

**Pull request merged**